index.js

```diff
@@ -18,100 +20,22 @@ module.exports = {
       },
 
       displayRevisions: function(context) {
-        if(!context.revisions) {
+        if(!context.revisions || context.revisions.length === 0) {
```

Comment:
I might still be asleep but I don't think we can do that
https://github.com/ember-cli-deploy/ember-cli-deploy-plugin/blob/master/index.js#L45